pages/packages/index.tsx

```diff
@@ -0,0 +1,22 @@
+import DocTemplate from '../docs/[...slug]'
+import { getDocProps } from '../../utils/docs/getDocProps'
+import { GetStaticProps } from 'next'
+import { GithubError } from 'next-tinacms-github'
+
+export const getStaticProps: GetStaticProps = async function(props) {
+  try {
+    return await getDocProps(props, 'index')
+  } catch (e) {
+    if (e instanceof GithubError) {
+      return {
+        props: {
+          previewError: { ...e }, //workaround since we cant return error as JSON
+        },
+      }
+    } else {
+      throw e
+    }
+  }
+}
+
+export default DocTemplate
```

Comment:
That's super interesting

Reply:
In a good way or a bad way?

Reply:
Good, I just didn't even think about re-using a template like that